test/rbs/rdoc/rbs_parser_test.rb

```diff
@@ -0,0 +1,396 @@
+require "test_helper"
+
+require "tempfile"
+
+require "rdoc/parser/rbs"
+
+class RBSParserTest < Test::Unit::TestCase
```

Comment:
Please update the class name and the file name.

```suggestion
class RDocPluginParserTest < Test::Unit::TestCase
```